scripts/round/setApplicationStatuses.ts

```diff
@@ -0,0 +1,85 @@
+// This script deals with updating a rounds projectMetaPtr.
+// Ideally this would be done via the UI and not this script
+import { ethers } from "hardhat";
+import hre from "hardhat";
+import { confirmContinue } from "../../utils/script-utils";
+import { roundParams } from "../config/round.config";
+import * as utils from "../utils";
+
+utils.assertEnvironment();
+
+const STATUS = {
+  PENDING: 0,
+  ACCEPTED: 1,
+  REJECTED: 2,
+  CANCELED: 3,
+};
+
+const applicationStatusBitMapIndex = 0;
+
+const projectIndexes: number[] = [0, 1, 2, 3];
+const statusArray: number[] = [
+  STATUS.PENDING,
+  STATUS.ACCEPTED,
+  STATUS.REJECTED,
+  STATUS.CANCELED,
+];
+
+const buildNewState = (
+  current: any,
```

Comment:
Nit: I would make this `BigInt` instead of `any`, and do the casting in the caller